src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -26,18 +30,71 @@
  */
 @WebServlet("/lecture")
 public class LectureServlet extends HttpServlet {
+
+  /* Used to create Entity and its fields */
+  private static final String LECTURE = "Lecture";
+  private static final String LECTURE_NAME = "lectureName";
+  private static final String VIDEO_URL = "videoUrl";
+  private static final String VIDEO_ID = "videoId";
+
+  /* Name of input field used for lecture name in lecture selection page. */
+  private static final String NAME_INPUT = "name-input";
+  /* Name of input field used for lecture video link in lecture selection page. */
+  private static final String VIDEO_INPUT = "video-input";
+  /* Default value if new lecture inputs are empty. */
+  private static final String DEFAULT_VALUE = "";
```

Comment:
Can we require these fields?  `.sendError` if one is empty, and add `required` to the input in html

Reply:
Added, keeping default value in case of an error